src/main/java/org/folio/consortia/service/impl/SharingSettingServiceImpl.java

```diff
@@ -43,13 +49,17 @@
 @Log4j2
 @RequiredArgsConstructor
 public class SharingSettingServiceImpl implements SharingSettingService {
+  private static final String NULL = "null";
```

Comment:
for null values better to use just as is without intducing constants

Reply:
fixed